term/theme.json

```diff
@@ -18,16 +37,95 @@
 		"typography": {
 			"fontFamilies": [
 				{
-					"fontFamily": "'HelveticaNeue-Light', 'Helvetica Neue Light', 'Helvetica Neue', Helvetica, Arial, 'Lucida Grande', sans-serif",
+					"fontFamily": "'HelveticaNeue Regular','HelveticaNeue-Light', 'Helvetica Neue Light', 'HelveticaNeue', Helvetica, Arial, 'Lucida Grande', sans-serif",
 					"name": "System Font",
 					"slug": "system-font"
 				}
+			],
+			"fontSizes": [
+				{
+					"size": "1rem",
+					"slug": "x-small"
+				},
+				{
+					"size": "1.25rem",
+					"slug": "small"
+				},
+				{
+					"size": "1.5rem",
+					"slug": "medium"
+				},
+				{
+					"size": "1.75rem",
+					"slug": "large"
+				},
+				{
+					"size": "2.375rem",
+					"slug": "x-large"
+				}
 			]
 		},
 		"useRootPaddingAwareAlignments": true
 	},
 	"styles": {
+		"blocks": {
+			"core/paragraph": {
+				"typography": {
+					"fontSize": "var(--wp--preset--font-size--medium)",
+					"textTransform": "capitalize"
+				}
+			}
+		},
+		"color": {
+			"background": "var(--wp--preset--color--custom-beige)",
+			"text": "var(--wp--preset--color--black)"
+		},
+		"elements": {
+			"h1": {
+				"typography": {
+					"fontSize": "64px",
+					"fontStyle": "normal",
+					"fontWeight": "700"
+				}
+			},
+			"h2": {
+				"typography": {
+					"fontSize": "48px"
```

Comment:
can any of these use preset values? if not, let's use rem values instead so they can scale